src/frequently-asked-questions-about-fsrs.md

```diff
@@ -0,0 +1,39 @@
+# Frequently asked questions about FSRS
+
+***
+
+Q1: Which platforms support FSRS?
```

Comment:
Is this really a FAQ? I don't think people ever inquire this.

Reply:
https://www.reddit.com/r/Anki/comments/1b89wt1/how_come_i_still_cant_download_the_fsrs_ankidroid/
https://www.reddit.com/r/Anki/comments/1b7x9ni/fsrs_on_different_versions_of_anki/
https://www.reddit.com/r/Anki/comments/1aocrce/is_it_advisable_to_keep_fsrs_on_since_ankidroid/
https://www.reddit.com/r/Anki/comments/1anl69o/is_fsrs_now_integrated_into_anki/
https://www.reddit.com/r/Anki/comments/1akzd2e/fsrs_and_ankidroid/
https://www.reddit.com/r/Anki/comments/1akphto/fsrs_on_ios/
https://www.reddit.com/r/Anki/comments/1aje203/ankidroid_update/
https://www.reddit.com/r/Anki/comments/1blvmhs/does_the_ankipc_settings_sync_with_ios_anki_and/
https://www.reddit.com/r/Anki/comments/1bkm88o/potential_issues_with_fsrs/
https://www.reddit.com/r/Anki/comments/1bufpkv/how_does_anki_keep_fsrs_updated/
https://www.reddit.com/r/Anki/comments/1bsgel6/is_it_possible_to_use_multiple_versions_of_anki/
https://www.reddit.com/r/Anki/comments/1bp1tli/how_can_i_anki_on_multiple_devices/
https://www.reddit.com/r/Anki/comments/1cph43b/switching_between_anki_versions_on_same_computer/
https://www.reddit.com/r/Anki/comments/1byy5c7/using_ankidroid_mess_up_fsrs/
https://www.reddit.com/r/Anki/comments/1e7h9ud/fsrs_vs_ankidroid/
https://www.reddit.com/r/medicalschoolanki/comments/1d21vyv/ankidroid_along_with_pc_anki_with_fsrs/
https://www.reddit.com/r/Anki/comments/19a259u/does_fsrs_sync_properly_between_ios_ankimobile/

I believe this answers your question (wink)

Reply:
are these still being asked after a year?

Reply:
Most of these are 5-8 months old. It's true that questions about FSRS support on different paltforms are asked less frequently these days, but I still feel like it's worth mentioning. Let's see what @dae thinks